### ISSUE TITLE
help

### ISSUE THREAD

**Issue by DreamH1gh** [google-research/bert#1304]
Why do I input the same batch and output different logits during debugging？
